src/flavors/duwamish_flavor/jstemplates/place-form.html

```diff
@@ -0,0 +1,91 @@
+
```

Comment:
You don't have to do it this time, I can also make a manual diff to see the changes, or I'll probably just merge it anyway :-)

Just for future reference, I wanted to point out that the structure of your commits should be considered. It helps for code review, and for troubleshooting bugs in the future.